.eslintrc

```diff
@@ -0,0 +1,207 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
+    "@typescript-eslint",
+    "sonarjs",
+    "security",
+    "github"
+  ],
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
```

Comment:
They add different ruleset? :) It's only a question.

Reply:
I don't know man. I saw you guys are using these plugins [here](https://github.com/asyncapi/modelina/blob/master/.eslintrc#L11) and I thought: After all... why not? :)

Reply:
TBH I don't remember 😅 Please remove the first one `plugin:@typescript-eslint/eslint-recommended` It should be the same ruleset.